StreamChatIntegrationSource/Base.lproj/Main.storyboard

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can we remove the storyboards? Less xml, less conflicts 😄

Reply:
Removed :)